components/objectKey/ObjectKeyHelper.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace app\components\objectKey;
+
+use app\models\Comment;
+use app\models\Extension;
+use app\models\File;
+use app\models\News;
+use app\models\Wiki;
+use yii\base\InvalidValueException;
+
+class ObjectKeyHelper
+{
+    const TYPE_NEWS = 'news';
```

Comment:
If class will be renamed to `ClassType`, these constants could be renamed to just `NEWS`, `WIKI` etc.

Reply:
Done.